### PR TITLE
add: Landing page #7

### LINKED ISSUE #7: Landing page

### PR DESCRIPTION
**Pull request by mradigen**
Closes #7 

Todo:
- [x] Timeline
- [x] Mobile view
~Footer~ (Designs not yet received, can add it in another PR)

### CODE REVIEW

File: src/app.html
Comment (KreativeThinker) [inline src/app.html:1]: Remove unnecessary font link

File: src/lib/components/Navbar.svelte [hunk 1]
```diff
@@ -29,16 +31,31 @@
...
+			<a href="{base}/">LUGVITC</a>
+			<a href="{base}/">Leaderboard</a>
+			<a href="{base}/">Projects</a>
+			<a href="{base}/">Dashboard</a>
+		</div>
+	</div>
+	<div class="hidden items-center space-x-4 xl:flex">
+		{#if $user.id}
+			<a href="{base}/">Welcome</a>
+			<a href="{base}/">{$user}</a>
```
Comment (KreativeThinker) [inline src/lib/components/Navbar.svelte]: Make user primary

File: src/lib/components/Navbar.svelte [hunk 2]
```diff
@@ -29,16 +31,31 @@
...
+		<div class="hidden items-center space-x-12 xl:flex">
+			<a href="{base}/">LUGVITC</a>
+			<a href="{base}/">Leaderboard</a>
+			<a href="{base}/">Projects</a>
+			<a href="{base}/">Dashboard</a>
+		</div>
+	</div>
+	<div class="hidden items-center space-x-4 xl:flex">
+		{#if $user.id}
+			<a href="{base}/">Welcome</a>
```
Comment (KreativeThinker) [inline src/lib/components/Navbar.svelte]: Change to sign out / logout

Comment (KreativeThinker) [reply src/lib/components/Navbar.svelte]: Make into a button

File: src/lib/components/Navbar.svelte (hunk 1 above)
Comment (KreativeThinker) [reply src/lib/components/Navbar.svelte]: Make into a button

**Review by KreativeThinker — CHANGES_REQUESTED** [review]

File: src/app.css
Comment (KreativeThinker) [inline src/app.css:1]: Reduce font sizes.

Comment (KreativeThinker) [reply src/app.css:1]: Replace FOSSit logo in navbar with lug logo

**Review by KreativeThinker — CHANGES_REQUESTED** [review]

**Comment by mradigen** [comment]
Ignore the deployment issues, the current deployment is viewable at https://mradigen.github.io/FOSSit24

**Review by KreativeThinker — CHANGES_REQUESTED** [review]
- [x] Make overflow-x hidden
- [x] Text still not showing in the two cards

**Review by KreativeThinker — APPROVED** [review]
Resolved

**Pull request merged**